[[2022-11-15]], [[2022-11-17]]

1. [[Genes and Proteins]]
	- [[Prokaryotic Transcription & Translation]]
2. [[Eukaryotic Cell]]
3. 3 Different RNA polymerases
	- RNA polymerase 1 transcribes [[DNA]]
	- RNA polymerase II transcribes mRNA and some snRNA
	- RNA polymerase III transcribes tRNA and some other small RNAs
	- Each RNA polymerase recognizes its own promoter
		1. Initiation of transcription
			- Genes turn on at different times
				1. When you are considered when genes get turned on, RNA polymerases must bind to promoters at right gene at right time for transcription and translation to occur. Leads to gene expression.
			- Requires a serios of transcription factors that assemble on a promoter to begin transcription
				1. *Sigma factor* = prokaryotic; *Series of transcription factors* = eukaryotes
				2. Necessary to get the RNA polymerase II enzyme to a promoter and to initiate gene expression
				3. Interacts w/ RNA polymerase to form initiation complex at promoter
		3. Termination
			- Hairpin loops
			- Termination sites not as well defined
4. Series of transcription factors
	- A generalized promoter of gene transcribed by RNA polymerase II is shown. Transcription factors recognize the promoter. RNA polymerase II then binds and forms the transcription initiation complex
	- TF indicates transcription factor
	- *Tatabox* is about 30 nucleotides upstream of code
		1. TFs bind to tatabox to indicate to RNA polymerase where to start
		2. To find tatabox, there is tata binding [[protein]] (TBP)
			- TBP can bind to tatabox
		3. First transcription factor is *TFIIB* (transcription factor two B)
		4. Last transcription factors is TFIIH (transcription factor two H)
			- TFIIH is a kinase
				1. Kinases phosphorylate other proteins
				2. TFIIH phosphorylates things to activate transcription
		5. Then RNA polymerase is in right spot and transcription begins
5. mRNA modifications (processing phase)
	- mRNA is produced in nucleus
	- Proteins are produced in cytoplasm
	- In eukaryotes, the primary transcript must be modified to become mature mRNA
		1. Make it ready to leave nucleus to go to cytoplasm
	- 3 modifications that must occur before it can leave
		1. Addition of 5' cap
			- Protection from degradation
			- Involved in translation initiation
			- Post transcriptional modification
		2. Addition of 3' poly-A-tail
			- Prokaryotes don't have this
			- Created by poly-A polymerase
			- Protection from degradation
			- If you have a longer tail, nucleases may "chew away" at tail but not reach code
				1. Length varies 
				2. Nucleases have exonuclease ability that tail protects from so RNA can stay around a little longer
			- Post transcriptional modification
		3. Removal of non-coding sequences (introns)
			- Pre-mRNA splicing done by spliceosome
			- Not all nucleotides code for ultimate [[protein]]
				1. Pieces don't code (*introns*) while other pieces do code (*exons*)
				2. Like patchwork of introns and exons
					- Introns = non-coding sequences
					- Exons = sequences that will be translated
				3. Ultimately must get exons together
					- Introns must be spliced out
					- There is well defined boundary between intron and exon w/ an A nucleotide at precise location. 
				4. Small ribonucleoprotein particles (snRNPs)
					- Responsible for removing introns by recognizing the intron-exon boundaries
					- snRNPs cluster w/ other proteins to form spliceosome
		4. When these three things are done, the mRNA is then called a mature mRNA
			- Ready to be packed and shipped out of nucleus 
			- When RNA is first transcribed, called primary RNA transcript
			- Mature mRNA after RNA processing
			- Has 5' cap, poly-A tail, and row of exons w/ no introns between
				1. Also has 5' UTR (untranslated region) and 3' UTR
					- These parts are not translated
					- Before 5' cap and 3' poly-a tail
						1. Poly-a tail is complimentary to u's in RNA or t's in [[DNA]] (allows us to isolate mature mRNAs for research purposes; can also allow us to detect cancerous cells)
							- BRAC1 expression correlates w/ breast [[cancer]]. Precedes even what mammogram can detect.
	- Alternative Splicing
		1. Single primary transcript can be spliced into different mRNAs by the inclusion of different sets of exons
			- Exons can be spliced in and out in different arrangements (resulting in different proteins)
			- Exome sequencing
			- Transcriptome refers to genes that are transcribed
		2. 15% of known [[human genetic disorders]] are due to altered splicing
			- 35-59% of human genes exhibit some form of alternative splicing
		3. Explains how 25000 genes of human genome can encode more than 80000 different mRNAs
		4. Different arrangements = different structures
			- Exons like lego blocks that can be arranged different ways (except that nucleic acids and proteins are very linear)
6. tRNA and Ribosomes
	- tRNA molecules carry [[amino acids]] to the ribosome for incorporation into a polypeptide
		1. Aminoacyl-tRNA synthetases add [[amino acids]] to the acceptor stem of tRNA
	- RNA has front end (5') and back end (3') and it is single stranded
		1. In [[DNA]] and RNA, hydroxyl group is at 3' end
		2. If you bring in nucleotide, you can create phosphodiester bond at that 3' hydroxyl group (little connecter for nucleotides)
		3. tRNA is like adapter
			- [[Amino acids]] can link at 3' end (ammino acid attachment site)
				1. Aminoacyl-tRNA synthetases attaches correct amino acid onto acceptor arm where 3' hydroxyl group is
				2. Once amino acid is attached, the tRNA is *charged* (ready to be used)
			- On other end, there is an anticodon
				- GAA
				- Anticodon is complementary to codon
	- [[Protein]] Synthesis Machinery - How do tRNAs and ribosomal RNAs work together?
		1. The ribosome has two primary functions 
			- Decode the mRNA
			- Form peptide bonds
		2. Ribosome has large subunit and small subunit, and it is composed of ribosomal RNA (rRNA)
			- AUG = start codon
			- --- Line up over start codon
			- tRNA carries methionine and its anticodon is complement to AUG. Lines up to mRNA. Once aligned, large ribosomal subunit comes in. Large ribosomal subunit has e, p, and a pockets that each preforms different function. A site is the accepter site. P site is polymerase site. E site is ejector site. Small ribosomal subunit on other side of mRNA than large subunit.
			- Nascent chain = newly formed polypeptide chain.
			- One tRNA that does not come into a site is initiator (methionine) tRNA. That is in p site. All others come in a site. Then ratcheting occurs. 
			- In that p site, polypeptide chain starts to grow.
		3. Translation begins when an initiator tRNA anticodon recognizes a codon on mRNA. The large ribosomal subunit joins the small subunit, and a second tRNA is recruited. As the mRNA moves relative to the ribosome, the polypeptide chain is formed. Entry of a resealed factor into the A site terminates translation and the components dissociate. 
		4. How do you make peptide bond?
			- Dehydration synthesis
			- NCC polypeptide chain from first amino acid to last amino acid. Formed as [[amino acids]] are linked
			- A peptide bond links the carboxyl end of one amino acid w/ amino end of another, expelling one [[water]] molecule. ---
		5. ---
		6. Elongation adds [[amino acids]]
			- 2nd charged tRNA can bind to empty A site
			- Requires elongation factor called EF-Tu to bind to tRNA and GTP
			- Peptide bond can then form
			- Addition of successive [[amino acids]] occurs as a cycle
		7. There are fewer tRNAs than codons
		8. Wobble pairing
			- Allows less stringent pairing between the 3' base of the codon and the 5' base of the anticodon
			- Fewer tRNAs are needed to accommodate all codons
		9. Termination
			- Elongation continues until the ribosome encounters a stop codon
			- Stop codons are recognized by release factors ---
7. [[Genes and Proteins]]