[[2022-11-15]]

1. Euchromatin vs Heterochromatin
	- Euchromatin is loosely packed 
		1. Transcriptionally active
		2. Form during interphase
			- [[DNA]] is replicated in S phase
			- DNA polymerase is enzyme that must be present in S phase
	- Heterochromatin is tightly packed
		1. Transcriptionally silent
		2. Form during mitosis
	- Chromatin is made up of genetic material and histone proteins
2. Transcription vs translation
	- Transcription = [[DNA]] --> RNA
		1. RNA polymerase is major transcription enzyme
		2. Occurs in the nucleus (of [[Eukaryotic Cell|eukaryotes]])
	- Translation = RNA --> protein
		1. Occurs in ribosomes in the cytoplasm or affixed to the rough endoplasmic reticulum 
	- Nucleolus is where ribosomes are created
		1. Extended loop is in euchromatin phase
			- Chromatin that extends outside of nucleolus
		2. Chromatin inside in rearranged in different ways depending on shape of nucleolus
			- Chromatin repositioning can lead to [[cancer]]
3. Central Dogma
	- First described by Francis Crick - the direction that [[DNA]] flows
		1. Information only flows from DNA --> RNA --> [[protein]]
		2. Transcription ([[DNA]] --> mRNA)
			- [[DNA]]-directed synthesis of RNA
			- Only template strand of DNA used
			- U (uracil) in RNA corresponds to T (thymine) in [[DNA]]
				1. RNA = AU,GC while DNA = AT,GC
			- messenger RNA (mRNA) used to direct synthesis of polypeptides 
		1. Translation (RNA --> [[protein]])
			- Synthesis of polypeptides
			- Takes place in ribosome
			- Requires several kinds of RNA
	- Retroviruses violate the flow of info by using enzyme reverse transcriptase to convert their RNA genome into DNA
		1. You can take mRNA and convert it back into DNA
			- Useful for research because RNA degrades more quickly than [[DNA]]
	- Instructions on DNA are transcribed onto mRNA. Ribosomes are able to read the genetic information inscribed on a strand of mRNA and use this information to string [[amino acids]] together into a [[protein]].
	- In RNA transcription, only a tiny piece of DNA opens up (no replication forks)
		1. Called transcriptional bubble
		2. One strand is genetic info for building protein (*the coding strand*)
		3. Other strand is called *the template strand*
			- RNA polymerase reads the template strand to make RNA copy of code w/ corresponding nucleotide bases
				1. IF DNA is TACG then RNA becomes AUGC
	- Once you transcribe mRNA, you have to process it
		1. When it's processed, it's recognized as ready to be shipped outside nucleus and used
	- After processing, mRNA is used in ribosomes to make proteins (translation)
4. RNA
	- All synthesized from DNA template by transcription
	- Major types
		1. Messenger RNA (mRNA)
			- Made by transcription
		2. Ribosomal RNA (rRNA)
			- Major component to a ribosome
			- More RNA in a ribosome than there is [[protein]]
			- rRNA is synthesized in the nucleolus 
		3. Transfer RNA (tRNA)
			- Like adapter molecule
			- There are 20 amino acids and 4 bases
				1. Bases must be read in groups of 3 to get 64 possible combinations
				2. Synonyms so that 64 combinations = 20 [[amino acids]]
			- One one end, has info related to [[nucleic acid]] but on other end has amino acid attached
				1. Used for translation
	- Other types
		1. Small nuclear RNA (snRNA)
		2. Signal recognition particle RNA
		3. Micro-RNA (miRNA)
	- RNA front = 5' and back = 3' (same as DNA)
5. Genetic Code
	- Francis Crick and Sydney Brenner determined how the order of nucleotides in [[DNA]] encoded amino acid order
	- Marshall Nirenberg identified the *codons* that specify each amino acid
		1. Codon = Block of 3 DNA nucleotides corresponding to an amino acid
			- Introduced single nucleotide insertions or deletions and looked for mutations
			- Several codons code for the same amino acid while others don't code for any amino acids
	- Stop codons = 3 codons used to terminate translation
		1. UUA
		2. UGA
		3. UAG
		4. Do not code for an amino acid
	- Start codon
		1. Codon (AUG) used to signify the start of translation
		2. Codes for methionine
	- Code is degenerate (some [[amino acids]] are specified by more than one codon)
		1. It often doesn't matter what base is in the third position
		2. Third position called wabble position if the third base doesn't matter (all codes for same amino acid)
			- Ex: valine
	- Code is universal (mitochondria and chloroplasts have some differences in "stop" signals)
		1. Works for both prokaryotes and eukaryotes
		2. Practically the same system across all organisms
	- Three reading frames
		1. When you read in different frame, you may create different amino acid
		2. Frameshift mutations (indicates importance of reading frame)
			- The deletion of two nucleotides shifts the reading frame of mRNA and changes the entire protein message, creating a nonfunctional [[protein]] or terminating protein synthesis altogether.
			- Changes the primary structure (which may impact the tertiary structure which may impact the function)
		3. To get in the right reading frame, you have to know where to begin
			- This is the importance of the start codon (AUG)
			- AUG codes for *methionine*
6. Structures of 20 [[amino acids]] found in proteins are composed of amino group (NH3), carboxyl group (COO) and a side chain. The side chain may be nonpolar, polar, or charged, as well as big or small.  The variety of amino acid side chains gives rise to the incredible variation of [[protein]] structure and function.
	- The characteristics of these amino acids are very important
	- Messing up an amino acid can change structure and function of proteins
7. Most of our [[DNA]] does not code for [[protein]] (just about 1%)
	- About 6.5 ft of [[DNA]]
	- Where do you start? How do you find starting point for ex hemoglobin?
8. [[Prokaryotic Transcription & Translation]]
9. [[Eukaryotic Transcription & Translation]]
10. [[Protein]] folding, modification, and targeting
	- Initiator tRNA
		1. Codes for methionine (AUG sequence)
		2. Every polypeptide chain must have methionine as 1st amino acid but not all proteins start w/ methionine. Must have info that directs where [[protein]] is supposed to go. As proteins are synthesized, front end has signaling sequence that fixes "zip code to package"
	- Signaling sequences
		1. Direct nascent proteins to be transported to correct cellular compartment
		2. Cleaved before protein is functional in post translational modifications
		3. One type of cystic fibrosis involves an incorrect signaling sequence
	- Post translational modifications
		1. May be necessary for [[protein]] to be activated
		2. Involve cutting off signaling sequence
		3. Insulin
			- Not automatically activated 
			- Many proteins are created in a pro form (pro insulin)
			- Portions of it are cleaved off to become activated version of insulin
	- Chaperone proteins
		1. Assist nascent (newly formed) proteins to fold in proper tertiary shape
			- Structure related to function
			- If pH or temp wrong, tertiary structure can be tweaked
				1. Chaperone proteins are like emergency response/ambulance proteins
				2. Also called heat shock proteins
				3. Can take misshapen [[protein]] and fix it
		2. Must first send [[protein]] to right place then ensure that it has right structure
			- Molecular repair system
		3. Mitochondria have inner and outer membrane
			- Proteins have to get through both inner and outer membrane w/ correct tertiary shape at end
			- These proteins are unfolded to get through membrane then refolded by chaperone proteins
	- Diseases
		1. Most of our diseases are linked to genetic mutations
		2. Category of diseases where there is not problem w/ [[DNA]] but *prion diseases*
			- No mutations in genetic info
			- New [[protein]] is folded into alternative 3d shape
			- Prion folds the [[protein]] differently then acts as chaperone and folds another protein incorrectly
			- Prion diseases = issue w/ misfolded diseases but no problem in [[DNA]]
			- Ex: mad cow disease