---
cards-deck: docs::BIO 111::Unit 3::Chapter 14
---

[[2022-11-02]]
^1667437601447

Avery, MacLeod, & McCarty, 1940s repeated Griffith’s experiments and discovered that ---::removal of protein did *not* destroy the ability of killed strains to transform live strains, but removal of DNA did destroy the transforming ability; Conclusion: the transforming material is most likely DNA
^1667437601454


Griffith mice experiment::Injected live S strand of bacteria (mice died) then injected live R strand of bacteria (mice lived); Injected boiled (dead) strand of S bacteria (mice lived); Injected dead S w/ live R (mice died); Concluded that deadly virulence must have passed from dead S to live R through *transformation*
^1667437601459

[[2022-11-04]]
Hershey and Chase (1950s)::Investigated bacteriophages (viruses that infect bacteria); The bacteriophage was composed of only DNA and protein; Question: Which of these molecules is the genetic material that is injected into the bacteria?
^1667609395810

Viruses are just --- packaged in ---::nucleic acid; protein coat; Not living but use cells to reproduce; Ex: bacteriophages infect bacterial cells
^1667609395823

Hershey and Chase labeled bacteriophage DNA w/ --- and bacteriophage protein w/ ---::radioactive phosphorus (P32); radioactive sulfur (S35); Proteins have sulfur as one of side chains in amino acids (Cystine has sulfur in its R group); No sulfur in nucleotides but do have phosphorous (no phosphorous in proteins unless they are phosphorylated)
^1667609395830

The radiation that comes from P32 is much higher than ---::S35; Allowed Hershey and Chase to track radioactive molecules of protein (w/ S35) and DNA (w/ P32)
^1667609395837

Hershey and Chase's experiment method::P32 went into [[DNA]]. Virus infects bacteria. As bacteria grows and replicates, radioactive virus builds inside bacteria. Centrifuged to separate phage from bacteria. Same process w/ S35-labeled protein. 
^1667609395844

Hershey and Chase discovered that only --- was found inside the bacteria after infection by virus::P32; Conclusion: DNA is the genetic material
^1667609395851

DNA is a --- and the building blocks of DNA are ---::nucleic acid; nucleotides (composed of 5-carbon sugar called deoxyribose, phosphate group (PO4), nitrogenous base (adenine, thymine, cytosine, guanine))
^1667609395859

DNA vs RNA::RNA has hydroxyl group and degrade very quickly; DNA does *not* have hydroxyl group (deoxyribose) and can stay around for ages; Both have phosphate group (negatively charged); Both have purines (A, G) and pyrimidine C but DNA has pyrimidine T while RNA has pyrimidine U
^1667609395867

Purines have a --- while pyrimidines have a --- structure::double ring; single ring; Purines = adenine and guanine; Pyrimidines = cytosine, thymine (DNA only), and uracil (RNA only)
^1667609395874

Nucleotides consist of ----::a nitrogenous base at the 1’ carbon, a phosphate group (-PO4) at the 5’ carbon, and a hydroxyl group (-OH) at the 3’ carbon; Front = 5' and back = 3'
^1667609395881

Nucleotides are polymerized to form nucleic acids using --- bonds::phosphodiester; Formed between the 5’ phosphate of one nucleotide and the 3’ hydroxyl of the next nucleotide; Each chain of nucleotides has a 5’ to 3’ orientation.
^1667609395889

DNA has repetitive sequence of --- and ---::ribose sugar; phosphate; Creates backbone; When you create these long chains, you bring in next nucleotide to 3' end where there is an available phosphate; Creates phosphodiester bond between 5' P of one nucleotide and 3' hydroxyl of next nucleotide
^1667609395896

Determining the 3-D structure of DNA involved the work of many scientists such as ---::Erwin Chargaff (determined "Chargaff's Rules")
^1667609395903

Chargaff's Rules::Amount of adenine = Amount of thymine (# A = # T); Amount of cytosine = Amount of guanine (# C = # G)
^1667609395911

Rosalind Franklin’s and Maurice Wilkins’ X-ray diffraction studies demonstrated that ---::DNA is helical, its diameter is 2 nm, and it makes a complete turn every 3.4 nm
^1667609395919

X-ray diffraction::Used by Rosalind Franklin and Maurice Wilkins to study DNA structure; Crystals have very regular, predictable structure so RF made crystal out of DNA then shot x-ray through crystal; Atoms made up of E's and nucleus w/ P's and N's; Some mass in electrons and most mass in nucleus and empty space between; If x-ray hits particles or nucleus, it deflects; Detector showed darkest areas where lots of x-rays went to same spot bc deflecting off same spots; RF used this info to measure distances and find shape of DNA
^1667609395927

James Watson and Francis Crick (1953)::One of most significant scientific advances of 20th century; Wanted to build model of DNA and fit pieces of earlier discoveries together; Deduced the structure of DNA using evidence from Chargaff, Franklin, and others
^1667609395935

James Watson and Francis Crick proposed a --- structure::double helix; Tried to pair different nucleotides w/ scale of 2 nm diameter; A to A did not equal 2 nm but A to T worked; Tried 5' to 5' but only 5' to 3' in double helix worked
^1667609395943

James Watson and Francis Crick's model deduced the --- of DNA::structure; When structure is known, we can deduce function; Led to questions such as how is DNA read?
^1667609395950

Critical DNA structure points::DNA is 2nm in diameter; 2 sugar-phosphate backbones in double helix; Bases toward the interior of one strand form H bonds w/ *complementary bases* on opposite strand
^1667609395958

DNA strands are ---::antiparallel; One oriented 5' to 3' (top to bottom) and other 3' to 5' (bottom to top); Strands wrap around each other to create the double helix
^1667609395965

--- can bind into major grooves on DNA, bending it and changing the diameter::Proteins; Normally 2nm in diameter, but change in DNA structure leads to change in function
^1667609395971

Gel Electrophoresis::DNA can be separated on the basis of size
^1667609395979

DNA Sequencing::Can deduce the actual base sequence in a piece of DNA; Frederick Sanger's dideoxy chain termination method (Dye-labeled dideoxynucleotides used to generate DNA fragments that terminate at different points. DNA is separated by electrophoresis, and from the order of fragments, the DNA sequence read using laser scanner)
^1667609395986

Meselson & Stahl (late 1950s) investigated DNA ---::replication, considering 3 possible mechanisms (conservative, semiconservative, dispersive)
^1667609395996

Conservative DNA replication theory::DNA opens up and creates a completely new copy (two old strands together and two new strands together in separate chain); Incorrect
^1667609396003

Semiconservative DNA replication theory::DNA opens up but partially stays together (one new strand + one old strand on each of the two separate chains); Correct
^1667609396010

Dispersive DNA replication theory::Pieces of the two separate strands of DNA go back and forth between new copy and old copy; Incorrect
^1667609396017

Meselson and Stahl (1958)::Bacterial cells were grown in a heavy isotope of nitrogen, N15 (radioactive version); Nitrogen is in nitrogenous bases (G, T, A, C), so all the DNA incorporated N15; Cells were then switched to media containing N14 (lighter version); DNA was extracted from the cells at various time intervals; The DNA from different time points was analyzed for ratio of N15 to N14 it contained
^1667609396024

Binary fission occurs once every 20 mins, so Meselson and Stahl extracted DNA at times ---::0 (when DNA heaviest bc of N15), then 20 (little lighter DNA), then 40 (still lighter), etc.
^1667609396031

Stahl and Meselson results::After 1 replication, the DNA consisted of a N14-N15 hybrid molecule; After 2 replications, half was a N14-N15 hybrid and other half was composed of N14 alone; Conclusion: the process of DNA replication is *semiconservative*
^1667609396039

Stahl and Meselson concluded that DNA replication was ---::semiconservative (when strand opens up, both strands used as template for a newly synthesized strand
^1667609396046

Each strand of DNA acts as --- for synthesis of new strand::template; Requires: Parental DNA strands, enzymes to do the copying, and nucleotide triphosphates to build the new strands; If strand has A, T must be created while if strand has G, complement must be C
^1667609396053

Initiation of DNA replication::Replication begins at an *origin*
^1667609396060

Elongation of DNA::New strands of DNA are synthesized by the enzyme *DNA polymerase* 
^1667609396067

Termination of DNA::Replication is terminated differently in prokaryotes (circular chromosomes) and eukaryotes (linear chromosomes)
^1667609396074

The chromosome of a prokaryote are ---::circular molecules of DNA; DNA polymerases bind to an *origin* and replication proceeds in both directions around the chromosome
^1667609396082

Characteristics of DNA at origin of replication::Lots of AT base pairs bc AT only has 2 H while GC base pairs have 3 H bonds (stronger); Makes DNA easier to open up
^1667609396089

DNA polymerase only works from --- to --- in one direction::5'; 3'; Nucleotide always added to 3' end as DNA elongates
^1667609396096

Replisomes::Group of proteins that do polymeration; DNA polymerase is critical enzyme in this group; Helicase; Primase enzyme
^1667609396103

Helicase (prokaryotic DNA replication)::Unzips the double helix bc DNA polymerase must only read one strand; The double helix is first unzipped by enzyme helicase
^1667609396111

Primase enzyme (prokaryotic DNA replication)::Adds a short RNA primer sequence after helicase unzips double helix; In order to copy, DNA polymerase must have a little primer (short piece of RNA); It is from this temporary primer that DNA polymerase creates new strand from
^1667609396117

DNA polymerase III (pol III) (prokaryotic DNA replication)::Enzyme adds nucleotides one at a time to the 3' end of the new (daughter) strand of DNA; Polymerase that does most of the work (can do 1000 bases a second in prokaryotes)
^1667609396124

Incoming nucleotides during DNA replication are ---::triphosphate; Two phosphates must be cleaved off to get energy to make process work; Nucleotides bind after two phosphates cleaved off
^1667609396131

DNA replication is semidiscontinuous and pol III can only add nucleotides to the ---::3’ end of the new strand; But, DNA strands are antiparallel; How do you copy other strand that is 5' to 3'?
^1667609396139

The *leading strand* of DNA is ---::synthesized continuously during replication (in the same direction that the replication fork is moving); But, The *lagging strand* is synthesized discontinuously creating *Okazaki fragments*
^1667609396146

[[2022-11-09]]
*Helicase* unzips DNA to get *replication fork.* *Pol III* can continuously follow leading strand w/ only one primer while lagging strand requires multiple fragments of primer. Then you go back and clean out the ---::temporary RNA primer
^1668038503491

In the replisome, helicase is like little machine that ---::opens up the DNA; Creates 3'-5' (lagging) strand and 5'-3' (leading) strand
^1668038503498

Primase creates the little bit of RNA to create doublestrandedness so ---::polymerase can start replication 
^1668038503504

While being replicated, the single strands innately want to come together w/ other strand (eukaryotes). However, we want strands to ---::stay apart during replication; Requires temporary insulation (or coat)
^1668038503510

Single stranded binding protein puts insulation on ---::each of the strands to prevent annealing (coming together w/ H bonds)
^1668038503516

RNA polymerase makes --- while DNA polymerase ---::makes primer; extends primer from 5'-3'
^1668038503523

DNA polymerase matches existing DNA bases w/ ---::complementary nucleotides and links them
^1668038503529

DNA polymerase::Adds new bases to 3' end of existing strands; Synthesizes in 5'-3' direction; Requires primer of RNA
^1668038503535

The enzymes for DNA replication are contained within the ---::replisome. The replisome consists of the primosome (composed of primase and helicase) and 2 DNA polymerase III (pol III) molecules
^1668038503541

Replication moves in one ---::direction, synthesizing both strands simultaneously.
^1668038503547

E. coli has 3 DNA polymerases, including ---::DNA polymerase I (pol I), DNA polymerase II (pol II), DNA polymerase III (pol III)
^1668038503553

DNA polymerase I (pol I)::Acts on lagging strand to remove primers and replace them with DNA; Reads doublestrandedness and cuts (w/ 5'-3' exonuclease ability) primer out to replace it w/ DNA; Also involved in profreeding (w/ 3'-5' exonuclease ability)
^1668038503559

DNA polymerase II (pol II)::Involved in DNA repair processes
^1668038503565

DNA polymerase III (pol III)::Main replication enzyme; Does most of copying
^1668038503571

All 3 --- have 3′-to-5′ exonuclease activity for ---::polymerases; proofreading; DNA pol I also has 5′-to-3′ exonuclase activity to remove primers
^1668038503578

Sliding clamp::Holds DNA polymerase into right position to copy as it moves down teh strand
^1668038503584

Single stranded binding proteins::Especially important for lagging strand; SSB; Wraps single-stranded DNA (ssDNA) with high affinity to protect it from degradation and prevent secondary structure formation
^1668038503590

RNA Primase::Adds temporary RNA primer for DNA polymerase to start replication
^1668038503596

There are --- DNA polymerases in total::three; Pol I, Pol II, and main replicator, Pol III
^1668038503602

There is a little nick (break) isn sugar phosphate backbones that DNA polymerase ---::cannot fix after primer; DNA ligase reads along doublestranded DNA and links broken areas of backbone back together; DNA ligase is final worker that completes synthesis
^1668038503608

Exonuclease::"ase" indicates enzyme; Works on nucleic acids; "Exo" indicates working on the end
^1668038503615

Endonuclease::Enzyme that can work on the middle of nucleotides
^1668038503621

--- was studied to learn about prokaryotic DNA replication::E coli
^1668038503628

The larger size and complex packaging of eukaryotic chromosomes means they must be replicated from ---::multiple origins (unlike prokaryotes); Can possible make DNA replication faster than that of binary fission 
^1668038503634

The enzymes of eukaryotic DNA replication are more complex than those of ---::prokaryotes. Not sequence specific; can be adjusted
^1668038503640

Initiation phase of replication requires more factors in eukaryotes to assemble both ---::helicase and primase complexes onto the template, then load the polymerase with its sliding clamp unit
^1668038503649

Primase in eukaryotes includes both ---::DNA and RNA polymerase
^1668038503656

Main replication polymerase in eukaryotes is a complex of ---::DNA polymerase epsilon (pol ε) and DNA polymerase delta (pol δ)
^1668038503663

Drosophila (fruit fly) has genome much bigger than bacteria but [[cell]] division only takes 8 mins bc of ---::multiple origins of replication on linear chromosome; Starts replicating at multiple places; One spot goes in one direction and other starts in opposite direction so that they meet up
^1668038503670

Overall process of DNA replication is similar in eukaryotes and prokaryotes except that chromosomes are ---::linear in eukaryotes and circular in prokaryotes
^1668038503676

Problem w/ linear chromosomes in eukaryotes::DNA polymerase needs primer to start copying. Removed primer after copying cannot be replaced; How do you copy ends of chromosomes (telomeres)? If you don't copy ends, chromosomes get shorter and shorter
^1668038503683

Solution to linear chromosomes in eukaryotes problem::Another enzyme is responsibel for restoring the ends of chromosomes
^1668038503689

Synthesizing the ends of the chromosomes in eukaryotes is difficult because of ---::the lack of a primer; With each round of DNA replication, the linear eukaryotic chromosome becomes shorter.
^1668038503695

Telomeres contain ---::repeated DNA sequence on the ends of eukaryotic chromosomes; Produced by telomerase enzyme; Also contains an RNA region that is used as a template so a DNA primer can be produced
^1668038503702

The enzyme telomerase builds extensions onto ---::3' overhang at end of leading strands; Critical to maintain ends of chromosomes
^1668038503708

Aging is associated w/ decreased associated ---::telomerase activity; Environment, lifestyle, and genetics all play a part as well
^1668038503715

Mistakes during DNA replication can lead to changes in the ---::DNA sequence and other DNA damage (*mutations*).
^1668038503721

DNA can also be damaged by chemical or physical agents called ---::*mutagens*; Repair mechanisms may be used to correct these problems.
^1668038503727

DNA replication all occurs during the --- phase of cell cycle::S (during interphase); Checkpoints are meant to access DNA and decide if repairable; Issues can occur in both prokaryotes and eukaryotes
^1668038503733

Specific DNA repair::targets a particular type of DNA damage; Ex: photorepair of *thymine dimers*
^1668038503740

Thymine dimers form in response to ---::UV light; Changes base diameter of DNA (no longer 2 nm; physical deformity); Sunburns --> skin cancer
^1668038503746

Photolyase::Identifies thymine dimers and breaks covalent bonds to restore DNAs' normal 2 nm width; Dimers that are not fixed cannot be read by DNA polymerase (copy error)
^1668038503752

Mismatch repair::Incorrect base is detected after replication; Mismatch repair enzymes detect and remove the base from the newly synthesized strand; The gap is filled with the correct base by DNA pol II
^1668038503758

How do mismatch repair enzymes know if incorrect base?::Unmethylated nucleotides are newly synthesized whereas original nucleotides are methylated
^1668038503765

Nonspecific DNA repair::Can repair many different kinds of damage; Ex: *excision repair* corrects damaged or mismatched bases by excising bad base and replacing it w/ good base
^1668038503770

Point mutations::Base substitutions (substitute one base for another); Transitions or transversions (also called missense mutations and results in amino acid substitution)
^1668038503777

Silent mutations::No effect on protein sequence (like synonyms)
^1668038503782

Nonsense mutations::Substitutes stop codon for an amino acid; Nonsense bc in place where stop codon should never be
^1668038503789

Frameshift mutations::Insertions or deletions of nucleotides may result in shift in reading frame or insertion of stop codon; Nucleotides read in groups of 3 so insertions or deletions may change groupings
^1668038503794

Chromosomal mutations::Changes in structure of chromosome; Can be deletions (part of chromosome is lost), duplication (part is copied), inversion (part is in reverse order), or translocation (part is moved to new location)
^1668038503801
