---
cards-deck: docs::BIO 111::Unit 4::Chapter 15
---

[[2022-11-16]]
Euchromatin (bio)::Loosely packed and transcriptionally; DNA form during interphase (replication occurs in S phase and DNA polymerase is enzyme that must be present in this phase)
^1668652005532

Heterochromatin (bio)::Tightly packed and transcriptionally silent; DNA form during mitosis
^1668690240317

Chromatin is made up of ---::genetic material and histone proteins
^1668690240328

Central Dogma::First described by Francis Crick - the direction that DNA  flows; Info only goes from DNA --> RNA --> protein
^1668690240335

Transcription::DNA --> mRNA; DNA-directed synthesis of RNA; Only template strand of DNA is used; U (uracil) in RNA corresponds to T (thymine) in DNA
^1668690240342

Translation::RNA --> protein; mRNA used to direct synthesis of polypeptides; Takes place in ribosome; Requires several kinds of RNA
^1668690240350

Retroviruses violate the flow of ---::info by using enzyme reverse transcriptase to convert their RNA genome info DNA; You can take mRNA and convert it back into DNA (useful for research bc RNA degrades more quickly than DNA)
^1668690240357

Instructions on DNA are transcribed onto mRNA. Ribosomes are able to read the ---::genetic information inscribed on a strand of mRNA and use this information to string amino acids together into a protein.
^1668690240365

In RNA transcription, only a tine piece of DNA ---::opens up (no replication forks); Called *transcription bubble*
^1668690240372

Coding strand::Strand that has genetic info for building protein; Other DNA strand is the *template strand* (RNA polymerase reads this to make RNA copy of code w/ corresponding nucleotide bases)
^1668690240381

RNA polymerase::Reads the template strand to make RNA copy of code w/ corresponding nucleotide bases; If template strand of DNA is TACG, then RNA becomes AUGC
^1668690240388

Once you transcribe mRNA, you have to ---::process it; When its processed, it's recognized as ready to be shipped outside nucleus and used
^1668690240395

After processing, mRNA is ---::used in ribosomes to make proteins (translation)
^1668690240402

RNA (bio)::All synthesized from DNA template by transcription; 2 major types (mRNA for transcription, rRNA for ribosomes, tRNA for translation)
^1668690258164

Messenger RNA::Made by transcription from DNA template strand
^1668690240414

Ribosomal RNA::Major component to ribosome; More RNA in a ribosome than there is protein; rRNA is synthesized in the nucleolus
^1668690240421

Transfer RNA (tRNA) (bio)::Like adapter molecule; 20 amino acids and 4 bases, so bases must be read in groups of 3 to get  64 possible combos (synonyms so that 64 combos = 20 amino acids); One one end, has info related to nucleic acid but on other has amino acid attached (translation)
^1668690258174

Other types of RNA::Small nuclear RNA (snRNA), Signal recognition particle RNA, Micro-RNA (miRNA)
^1668690240431

RNA front = --- and back = ---::5'; 3' (same as DNA)
^1668690240438

[[2022-11-17]]
Genetic Code History::Francis Crick and Sydney Brenner determined how the order of nucleotides in DNA encoded amino acid order; Marshall Nirenberg identified the codons that specify each amino acid
^1668707983435

Codon::Block of 3 DNA nucleotides corresponding to an amino acid; Introduced single nucleotide insertions or deletions and looked for mutations; Several codons code for the same amino acid while others don't code for any amino acids
^1668707983448

Stop codons::Three codons used to terminate translation (UUA, UGA, UAG); Do not code for an amino acid
^1668707983456

Start codon::Codon (AUG) used to signify the start of translation; Codes for methionine
^1668707983463

Code is degenerate, meaning that ---::some amino acids are specified by more than one codon; It often doesn't matter what base is in the third position; Third position called *wabble position* if the third base doesn't matter (all codes for same amino acid; ex valine)
^1668707983470

Wabble position::If the third base doesn't matter (the first two bases all code for the same amino acid), then it is called the wabble position (ex valine)
^1668707983477

Code is universal::Mitochondria and chloroplasts have some differences in "stop signals"; Code works for both prokaryotes and eukaryotes and practically the same system across all organisms
^1668707983484

Three reading frames::When you read in different frame, you may create different amino acid; Frameshift mutations indicate importance of reading frame
^1668707983492

Frameshift mutations (bio)::The deletion of two nucleotides shifts the reading frame of mRNA and changes the entire protein message, creating a nonfunctional protein or terminating protein synthesis altogether; Changes primary structure (which may impact the tertiary structure which may impact the function)
^1668707997511

To get in the right reading frame, you must know ---::where to begin; This is the importance of the start codon (AUG); AUG codes for *methionine*
^1668707983499

Structures of 20 amino acids found in proteins are composed of ---::amino group (NH3), carboxyl group (COO), and a side chain (could be nonpolar, polar, or charged as well as big or small); The variety of amino acid side chains gives rise to the incredible variation of protein structure and function
^1668707983507

Most of our DNA does not code for ---::proteins (just about 1%); About 6.5 ft of DNA total, so where do you start? How do you find starting point for a specific protein (ex hemoglobin)?
^1668707983514

Transcription requires::promoter (forms recognition and binding site for RNA pol), start site (tatabox w/ sigma factors for prokaryotes and series of TFs for eukaryotes), elongation process (5'-3' direction using RNA pol and transcription bubble), and termination site (stop sequence or hairpin loops)
^1668707983521

The promoter, start site, and termination site form a ---::transcription unit; Transcription makes RNA version of coding strand from template strand
^1668707983528

Prokaryotic start site::How do you find correct position to start?::Certain signals indicate starting point; Upstream of transcription start site is region of Ts and As (called *tatabox*); Proteins known as *sigma factors* identify tataboxes in prokaryotes
^1668707983535

Sigma factors::Identify tataboxes in prokaryotes to start transcription; The sigma subunit of prokaryotic RNAO polymerase recognizes consensus sequences found in the promoter region upstream of the transcription start sight; The sigma subunit dissociates from the polymerase after transcription is complete
^1668707983543

Transcription elongation process::Grows in the 5'-3' direction as ribonucleotides are added; Transcription bubble = contains RNA polymerase, DNA template, and growing RNA transcript; After transcription bubble passes, the now transcribed DNA is rewound as it leaves the bubble
^1668707983550

Termination site::Marked by sequence that signals "stop" to polymerase; Causes the formation of phosphodiester bonds to cease; RNA-DNA hybrid within the transcription bubble, so RNA pol must release the DNA and allow DNA to rewind; Hairpin loops caused by the binding of RNA can force termination
^1668707983557

Prokaryotic transcription is coupled to ---::translation; mRNA begins to be translated before transcription is finished; Front end of protein is called *N terminal* (corresponds to 5' end) and back end is *C terminal* (corresponds to 3' end)
^1668707983565

As soon as the front end of prokaryotic RNA has been created after starting transcription, translation can start to occur as ---::ribosomes bind and bacteria can make protein before RNA is fully synthesized
^1668707983573

Prokaryotes can code for multiple proteins on one ---::mRNA;  If there is one mRNA w/ multiple genes, it is called an *operon* (unique to prokaryotes)
^1668707983580

Operon::Seen in prokaryotes; Grouping of functionally related genes; Multiple enzymes for a pathway can be regulated together; One mRNA w/ multiple genes
^1668707983587

In eukaryotes, there are three different ---::RNA polymerases; RNA pol I transcribes rRNA, RNA pol II transcribes mRNA and some snRNA, and RNA pol III transcribes tRNA and some other small RNAs; Each RNA pol recognizes its own promoter
^1668707983595

[[2022-11-21]]
RNA pol I::Transcribes rRNA
^1669070714286

RNA pol II::Transcribes mRNA and some snRNA
^1669070714295

RNA pol III::Transcribes tRNA and some other small RNAs
^1669070714301

Each RNA pol recognizes its own ---::promoter; Allows for the initiation of transcription through a series of transcription factors in eukaryotes
^1669070714308

Series of transcription factors are necessary to get the ---::RNA pol II enzyme to a promoter and to initiate gene expression; Interacts w/ RNA pol to form initiation complex at promoter
^1669070714314

Genes turn on at different times. When genes get turned on, RNA pol must bind to promoters at correct ---::time for transcription and translation to occur; Leads to gene expression
^1669070714321

Sigma factor initiates transcription in ---::prokaryotes; Series of TFs in eukaryotes
^1669070714327

Termination of transcription::Sites not well defined in eukaryotes; Hairpin loops can force termination
^1669070714334

Series of TFs (eukaryotes)::Transcription factors recognize promoters; RNA pol II then binds and forms the transcription initiation complex
^1669070714340

Tatabox::About 30 nucleotides upstream of code; TFs bind to tatabox to indicate to RNA pol where to start; To find tatabox, there is tata binding protein (TBP)
^1669070714347

Tata binding protein::Bind to tataboxes to indicate to RNA pol II where to start transcription
^1669070714353

TFIIB (transcription factor two B)::First transcription factor
^1669070714360

TFIIH (transcription factor two H)::Last transcription factor; Kinase (phosphorylates things to activate transcription)
^1669070714366

mRNA modifications (processing phase)::Addition of 5' cap and 3' poly-A tail; Removal of non-coding sequences (introns)
^1669070714372

Addition of 5' cap in processing phase::Added to mRNA to protect it from degradation; Involved in translation initiation; Post transcriptional modification
^1669070714378

Addition of 3' poly-A tail in processing phase::Created by poly-A polymerase; Protection from degradation; If you have a longer tail, nucleases may "chew away" at tail but not reach code (length varies but allows RNA to stay around a little longer); Post transcriptional modification
^1669070714385

Removal of non-coding sequences in processing phase::Pre-mRNA splicing done by spliceosomes; Removal of introns leaving only exons
^1669070714391

Exons::Sequences in mRNA that will be translated; Left during the processing phase (introns are spliced out)
^1669070714398

Small ribonucleoprotein particles (snRNPs)::Responsible for removing introns during the processing phase of mRNA; Recognizes the intron-exon boundaries; snRNPs cluster w/ other proteins to form spliceosomes
^1669070714404

mRNA is produced in the ---::nucleus; Proteins are produced in cytoplasm; In eukaryotes, the primary transcript must be modified to become mature mRNA (makes it ready to leave nucleus to go to cytoplasm)
^1669070714410

After the processing phase, mRNA is called::mature mRNA; Ready to be packaged and shipped out of nucleus (when mRNA first transcribed, called primary RNA transcript); Has 5' cap, poly-A tail, and row of exons w/ no introns between
^1669070714417

In addition to 5' cap, poly-A tail, and exon row, mature RNA has 5' UTR (untranslated region) and 3' UTR region; These parts not translated; Before the cap and tail

Poly-A tail is complimentary to ---::Us in RNA or Ts in DNA; Allows us to isolate mature mRNAs for research purposes; Can also allow us to detect cancerous cells (BRAC1 expression correlates w/ breast cancer)
^1669070714423

Alternative splicing::Single primary transcript can be spliced into different mRNAs by the inclusion of different sets of exons; Exons can be spliced in and out different arrangements (resulting in different proteins); Exome sequencing; Transcriptome refers to genes that are transcribed
^1669070714429

15% of known human genetic disorders are due to ---::altered splicing; 35%-59% of human genes exhibit some form of alternative splicing
^1669070714436

Alternative splicing explains how 25000 genes of human genome can encode the more than ---::80000 different mRNAs; Different arrangements = different structures; Exons like lego blocks that can be arranged in different ways (except that nucleic acids and proteins are very linear)
^1669070714442

tRNA molecules carry amino acids to the ---::ribosome for incorporation into a polypeptide; Aminoacyl-tRNA synthetases add amino acids to the acceptor stem of tRNA
^1669070714448

RNA has front end (5') and back end (3') and it is single stranded; If you; The --- group is at the 3' end::hydroxyl group; In DNA and RNA, you can create phosphodiester bonds at that 3' hydroxyl group (little connecter for nucleotides)
^1669070714454

tRNA is like an ---::adapter; Amino acids can link at 3' end (ammino acid attachment site) w/ aid of aminoacyl-tRNA synthetases; Once amino acid is attached, the tRNA is *charged* (ready to be used); On other end, there is anticodon
^1669070714461

Anticodon loop on tRNA contains three nucleotides complementary to ---::mRNA codons
^1669070714467

Each aminoacyl-tRNA synthetase recognizes only ---::one amino acid but several tRNAs; Adds amino acids to acceptor stem of tRNA
^1669070714473

Charged tRNA::tRNA that has an amino acid added using energy from ATP; Can undergo peptide bond formation without additional energy; Ribosomes do not verify amino acid attached to tRNA
^1669070714479

Protein synthesis machinery::How do tRNAs and ribosomal RNAs work together? Ribosome has two primary functions (decode mRNA and form peptide bonds)
^1669070714485

The ribosome has multiple ---::tRNA binding sites; P site - binds to tRNA attached to growing peptide chain; A site - binds the tRNA carrying the next amino acid; E site - binds the tRNA that carried the last amino acid
^1669070714491

Peptydyl transferase::Enzymatic component of the ribosome; Forms peptide bonds between amino acids to help w/ translation 
^1669070714498

Ribosome has large subunit and small subunit, and it is composed of ---::ribosomal RNA (rRNA)
^1669070714504

In prokaryotes, the initiation complex includes ---::Initiator tRNA charged w/ N-formylmethionine, small ribosomal subunit, and mRNA strand
^1669070714511

Ribosome binding sequence (RBS) of mRNA positions ---::small subunit correctly before large subunit is added; Beginning of translation process; Initiator tRNA bound to p site w/ A site empty in prokaryotes
^1669070714519

Initiations in eukaryotes similar to prokaryotes except ---::Initiating amino acid is methionine, more complicated initiation complex, and lack of an RBS (small subunit that binds to 5' cap of mRNA)
^1669070714526

Translation begins when an initiator tRNA anticodon recognizes a 
---::codon on mRNA. The large ribosomal subunit joins the small subunit, and a second tRNA is recruited. As the mRNA moves relative to the ribosome, the polypeptide chain is formed. Entry of a release factor into the A site terminates translation and the components dissociate.
^1669070714532

AUG::The start codon of translation; Codes for methionine; tRNA carries meth and binds to mRNA w/ the anticodon (UAC); Then subunits line up on opposite sides of mRNA strand
^1669070714539

Large ribosomal subunit has ---::E, P, and A pockets that each preform different function; A site = acceptor site for amino acids during translation; P site = polymerase site; E site = ejector site
^1669070714545

Nascent chain::Newly formed polypeptide chain from translation
^1669070714551

One tRNA that does not come into the A site is the ---::initiator (methionine) tRNA; That is bound in P site; All other tRNAs come in the A site; Then ratcheting occurs down the line of mRNA (in the P site, the polypeptide chain starts to grow)
^1669070714558

How do you make peptide bond?::Dehydration synthesis; NCC polypeptide chain from first amino acid to last amino acid; Formed as amino acids are linked; A peptide bond links the carboxyl end of one amino acid w/ amino end of another, expelling one water molecule in the process
^1669070714564

The elongation process of translation adds ---::amino acids; 2nd charged tRNA can bind to empty A site; Requires elongation factor called EF:-Tu to bind to tRNA and GTP so peptide bond can form; Addition of successive amino acids occurs as a cycle
^1669070714570

There are fewer --- than codons::tRNAs; Wobble pairing allows less stringent pairing between the 3' base of the codon and the 5' base of the anticodon; Fewer tRNAs are needed to accommodate all codons
^1669070714576

Termination of translation::Elongation continues until the ribosome encounters a stop codon; Stop codons are recognized by release factors, which release the polypeptide from the ribosome
^1669070714582

Initiator tRNA::Codes for methionine (AUG sequence); Every polypeptide chain must have meth as 1st amino acid but not all proteins start w/ meth; Must have info that directs where protein is supposed to go; As proteins are synthesized, front end has signaling sequence that fixes "zip code to package"
^1669070714589

Signaling sequences::Direct nascent proteins to be transported to correct cellular compartment; Cleaved before protein is functional; One type of cystic fibrosis involves an incorrect signaling sequence (wrong "zip code")
^1669070714596

Post translational modifications::May be necessary for nascent protein to be activated; Involve cutting off signaling sequence; Ex insulin not automatically activated (many proteins created in pro form (pro insulin) and portions are cleaved off to become activated version of insulin)
^1669070714602

Chaperone proteins::Assist nascent proteins to fold in proper tertiary shape; Structure related to function (ip ph or temp wrong, tertiary structure may be tweaked so chaperone proteins are like emergency response proteins); Also called heat shock proteins
^1669070714609

Chaperone proteins can take --- and fix them::misshapen proteins; Like ambulance proteins
^1669070714616

We must first send proteins to right place through signaling sequences then ---::ensure that it has right structure; Molecular repair system (chaperone proteins)
^1669070714622

Mitochondria have inner and ---::outer membrane; Proteins have to get through both inner and outer membrane w/ correct tertiary shape at end; Those proteins are unfolded to get through membrane then refolded by chaperone proteins
^1669070714631

Most of our diseases are linked to ---::genetic mutations; Category of diseases where there is not problem w/ DNA (called *prion diseases*)
^1669070714637

Prion disease::Not caused by mutations in genetic info; New protein is folded into alternative 3d shape; Prion folds the protein differently then acts as chaperone and folds another protein incorrectly; Issue w/l misfolded proteins but no problem in DNA; Ex: mad cow disease
^1669070714644



