---
cards-deck: docs::BIO 111::Unit 4::Chapter 16
---

[[2022-11-24]]
The control of which genes are expressed dictates whether a cell is ---::an eye cell or liver cell or something else; The genetic content of each somatic cell is same but not all genes expressed in every cell
^1669442089640

It is the differential gene expression patterns that arise in different cells that give rise to ---::a complete organism; Groups of cells w/ coordinated function --> tissues and groups of tissues --> organs
^1669442089650

Three critical steps to controlling gene expression::Access to DNA, Transcription, and Translation
^1669442089657

Accessing the DNA::If DNA is tightly bound, you can't read instructions; Must read nucleotides in order; Heterochromatin vs Euchromatin
^1669442089664

Controlling gene expression is often accomplished by controlling ---::transcription initiation (especially in prokaryotes); Regulatory proteins bind to DNA (may block or stimulate transcription)
^1669442089671

Transcription involves --- while translation involves ---::converting DNA to mRNA; converting RNA to protein
^1669442089678

You can have proteins that body has produces but not ---::turned on yet; Ex: caspases proteins = associated w/ apoptosis (programmed cell death)
^1669442089685

Body has already produced caspases proteins in an ---::inactivated form (called pro-caspases) but they must be activated for apoptosis to occur (then become caspases)
^1669442089692

Regulatory proteins::Bind to DNA (this is first step to accessing DNA); Many of these are transcription factors (may block or stimulate DNA access)
^1669442089698

Prokaryotes regulate gene expression in response to their ---::environment; If bacteria needs amino acid tyrosine to build proteins, can either get in from environment or create it (picking it up is easier than making it) 
^1669442089705

Eukaryotes regulate gene expression to maintain ---::homeostasis in the organism; If you don't drink, you are thirsty. If you are hot, you sweat. If it's cold, you shiver. All processes meant to maintain a "steady state"; Changing genes can effect homeostasis as well
^1669442089712

Most DNA-binding proteins bind in the --- of the double helix::major groove; DNA is 2 nm in diameter, so when proteins bind, you bend DNA (local distortion); Proteins w/ their R groups can bind to groove w/ H bonds
^1669442089718

DNA-Binding Motifs::Regions of regulatory proteins that bind to DNA
^1669442089724

Functional groups stick off base pairs and are each ---::functionally different; Physical features that define AT base pair vs GC base pair; Those different features face the major groove; Proteins w/ their R groups can bind to the major groove and temporarily form H bonds
^1669442089731

Different characteristic/components of DNA binding proteins::Can determine protein based on structural differences; Can identify unknown proteins based off these features; If it has one of these characteristics, it is DNA binding protein
^1669442089738

Helix-turn-helix motif::Also called homeodomain motif; Domain = region of protein; Region that has two alpha helixes (One lies directly in major groove and other is perp to groove)
^1669442089745

Zinc finger motif::Another distinct characteristic that can be used to identify DNA binding proteins
^1669442089752

Leucine zipper motif::Characteristic of DNA binding proteins when the alpha helixes are in adjacent major grooves
^1669442089759

Prokaryote gene regulation and properties::Lack nucleus; DNA found in cytoplasm; Transcription and Translation occur nearly simultaneously; Regulation primarily at level of transcription (If you control transcription, you control translation)
^1669525951287

Eukaryote gene regulation and properties::Contain nucleus; DNA confined to nucleus; Transcription occurs prior to translation (which only occurs in cytoplasm); Regulation at many levels (access to DNA, transcription, translation; post-translation)
^1669525951299

Primary transcript::RNA that has just been produced (unprocessed); Processing turns primary transcript RNA into mature RNA (Must methylate, cut out introns, add poly-A tail, and add 5' cap)
^1669525951307

[[2022-11-26]]
Prokaryotic cells often respond to their environment by changes in ---::gene expression
^1669525951316

Operons::When prokaryotic genes involved in the same metabolic pathways are organized; Only found in prokaryotes; Regions on chromosomes that regulate for similar proteins
^1669525951325

Prokaryotes have proteins that activate transcription and proteins that ---::repress transcription; Induction and Repression; Positive control and Negative control 
^1669525951333

Prokaryote Induction::Enzymes for a certain pathway that are produced in response to a substrate
^1669525951342

Prokaryote Repression::Capable of making an enzyme but does not control transcription initiation 
^1669525951349

Positive control::Increases frequency of prokaryotic initiation of transcription; Activators enhance binding of RNA pol to promoter; Effector molecules can enhance or decrease transcription
^1669525951358

Negative control::Decreases frequency of prokaryotic transcription; Repressors bind to operators in DNA; Allosterically regulated; Responds to effector molecules (enhances or abolishes binding to DNA)
^1669525951367

TRP operon::Prokaryote genes for the biosynthesis of tryptophan; Operon is not expressed when the cell contains sufficient amounts of tryptophan; Operon is expressed when levels of tryptophan are low
^1669525951374

TRP repressor is a ---::helix turn helix protein (DNA binder protein) that binds to the operator site located adjacent to the trp promoter; TRP operon is negatively regulated by the trp repressor protein; When it binds, RNA pol does not have normal landing place
^1669525951383

trp repressors bind to operators to ---::block transcription of trp operon in prokaryotes; Binding of the repressor to the operator requires a corepressor (which is tryptophan); Low levels of tryptophan prevent the repressor from binding to the operator 
^1669525951391

The five genes that are needed to synthesize tryptophan in E. coli are located next to each other in the ---::trp operon. When tryptophan is plentiful, two tryptophan molecules bind the repressor protein at the operator sequence. This physically blocks the RNA polymerase from transcribing the tryptophan genes. When tryptophan is absent, the repressor protein does not bind to the operator and the genes are transcribed; 5 start codons and 5 stop codons for 5 different proteins
^1669525951398

Glucose repression::Organisms prefer to use glucose in the presence of other sugars; Mechanism involves activator protein that stimulates transcription; CAP has cAMP as effector 
^1669525951405

Catabolic activator protein (CAP)::An allosteric protein w/ cAMP as effector; If you cleave of phosphate of ATP, you get ADP. If you cleave off another, you get AMP. If you are really low on energy, cAMP is created
^1669525951413

Level of cAMP in cells is reduces in presence of ---::glucose so that no stimulation of transcription from CAP-responsive operons takes place; cAMP means that cell is very low on energy and cell must get glucose into cell to reduce level of cAMP; cAMP binds to allosteric site of CAP and changes its shape (this induces transcription so cell can use other sugars)
^1669525951421

Inducer exclusion::The presence of glucose inhibits the transport of lactose into cell; Cells prefer to use glucose other all other sugars
^1669525951428

When glucose levels fall, E. coli may use other sugars for fuel but must ---::transcribe new genes to do so. As glucose supplies become limited, cAMP levels increase. This cAMP binds to the CAP protein, a positive regulator that binds to an operator region upstream of the genes required to use other sugar sources
^1669525951436

LacZ operon uses lactose so when cAMP + CAP binds to mRNA, RNA pol is activated and ---::transcripts operons such as lacZ that allows cell to get more energy; If glucose used up, cell can take lactose and other sugars
^1669525951442

Lac operon::Negatively regulated by a repressor protein; Lac repressor binds to operator to black transcription; In the presence of lactose, an inducer molecule (allolactose) binds to repressor protein; Repressor can no longer bind to operator; Transcription proceeds
^1669525951450

Even in the absence of lactose, the lac operon is ---::expressed at a very low level; Contains genes for the use of lactose as an energy source (B-galactosidase (lacZ), permease (lacY), and transacetylase (lacA)); Gene for lac repressor (lacI) is linked to the rest of the lac operon
^1669525951457

Transcription of the lac operon is carefully regulated so that its expression only occurs when ---::glucose is limited and lactose is present to serve as an alternative fuel source; Decisions must be made depending on resources
^1669525951464

Eukaryotic chromatin structure is directly related to ---::control of gene expression; DNA wound around histone proteins to form nucleosomes (no nucleosomes in prokaryotes); Nucleosomes may block access to promoter
^1669525951472

IT is possible to transcribe ---::nucleosome region of DNA (can read it while wrapped around histone protein) but histones can be modified to result in greater condensation (modifications either allow access to DNA or they don't depending on methylation)
^1669525951479

Methylation::Thought to play a major role in gene regulation; Access to DNA may depend on methylation; Many inactive mammalian genes are methylated; Lessor role in blocking accidental transcription of genes turned off
^1669525951486

Modifying histones correlated w/ ---::active verses inactive regions of chromatin; Can be methylated - found in inactive regions; Can be acetylated - found in active regions
^1669525951505

Methyl group::-CH3; Found in inactive regions of histones bc too small to create enough space for histones to be read; Forms heterochromatin
^1669525951515

Acetyl group::CH2CH3; Acetyl groups are bigger than methyl groups; If two carbons, acetyl group; If one carbon, methyl group (-CH3); Histones that are acetylated are accessable bc provides necessary space for DNA to be read; Forms Euchromatin (more spread out bc it has more carbons)
^1669525951522

DNA itself can be methylated as well as ---::histones; Original strands of DNA are methylated whereas newly synthesized DNA strands are not; This is how cell knows if DNA is wrong and can correct newly synthesized DNA in replication process
^1669525951529

DNA is folded around histone proteins to create nucleosome complexes. These nucleosomes control the access of ---::proteins to the underlying DNA. When viewed through an electron microscope, the nucleosomes look like beads on a string.
^1669525951537

Nucleosomes can slide along DNA. When nucleosomes are spaced closely together, transcription factors cannot ---::bind and gene expression is turned off. When the nucleosomes are spaced far apart, the DNA is exposed. Transcription factors can bind, allowing gene expression to occur. Modifications to the histones and DNA affect nucleosome spacing.
^1669525951543

Methylation of DNA and histones causes nucleosomes to ---::pack tightly together; Transcription factors cannot bind to DNA and genes are not expressed 
^1669525951551

Histone acetylation results in loose packing of ---::nucleosomes; Transcription factors can bind to DNA and genes are expressed 
^1669525951558

Histone proteins and DNA nucleotides can be modified chemically. Modifications affect ---::nucleosome spacing and gene expression.
^1669525951566

Epigenetic changes to chromatin may result from ---::development, environmental chemicals, drugs/pharmaceuticals, aging, diet; You accumulate more methylation over time based on your lifestyle; Environment puts meth tags on DNA (keeps some genes from being accessed and allows other to be accessed)
^1669525951573

Epigenetic changes may result in ---::cancer (uncontrolled cell growth), Autoimmune diseases, Mental disorders, Diabetes; DNA methylation and chemical modification of histone tails alter spacing of nucleosomes and change gene expression
^1669525951579

Histone acetylases::Some coactivators have been shown to be histone acetylases
^1669525951586

Control of transcription is more complex in ---::eukaryotic regulation; Major differences from prokaryotes (Eukaryotes have DNA organized into chromatin; Complicated protein-DNA interaction; Eukaryotic transcription occurs in nucleus; Amount of DNA involved in regulating eukaryotic genes is much larger)
^1669525951593

Transcription is increased in eukaryotes by removing ---::higher order chromatin structure that would prevent transcription; Histone code postulated to underlie the control of chromatin structure
^1669525951601

Chromatin remodeling complexes::Large complex of proteins; Modify histones and DNA; Also change chromatin structure
^1669525951608

ATP dependent chromatin remodeling factors::Function as molecular motors; Catalyze four different changes in DNA/histone binding; Makes DNA more accessible to regulatory proteins
^1669525951617

Eukaryotic general transcription factors::Necessary for the assembly of a transcription apparatus and recruitment of RNA pol II to a promoter; TFIID recognizes tatabox sequences
^1669525951623

Eukaryote specific transcription factors::Increase the level of transcription in certain cell types or in response to signals
^1669525951630

Enhancer::DNA sequence that promotes transcription (eukaryotes); Upstream of where code that is transcribed is; Interacts w/ proteins known as activators that interact w/ transcriptional machinery
^1669525951637

Each enhancer (promotes DNA transcription) is made up of short DNA sequences called ---::*distal control elements*; This  interaction creates a region that facilitates transcription of one gene but may limit transcription of others (crowding out)
^1669525951645

Activators::Bound to distal control elements of enhancers and interact w/ mediator proteins and TFs; Two different genes may have same promoter but different distal control elements, enabling differential gene expression
^1669525951652

Nearly every eukaryotic gene represents a unique case of ---::enhancers and activators; Greater flexibility to respond to many signals; Virtually all genes that are transcribed by RNA pol II need the same suite of general factors to assemble an initiation complex
^1669525951658

General transcription factors::Necessary for the assembly of transcription apparatus and recruitment of RNA pol II to a promoter; *TFIID* recognizes tatabox sequences
^1669525951666

Specific transcription factors::Increase the level of transcription in certain cell types or in response to signals
^1669525951673

Promoters form the binding sites for ---::general transcription factors; Mediate the binding of RNA pol II to the promoter
^1669525951680

Enhancers are the binding site of ---::specific transcription factors; DNA bends to form loop to position enhancer closer to promoter
^1669525951687

[[2022-11-30]]
Control of gene expression in eukaryotes usually involves the control of ----::transcription initiation; Gene expression can also be controlled after transcription w/ alternative splicing, mRNA degradation, and miRNA
^1669869733361

Mature mRNAs contain ---::5' G-cap, 3' poly-A tail, and introns spliced out
^1669869733374

Alternative splicing (bio)::Introns are splicing out of pre-mRNAs to produce the mature mRNA; Tissue specific alternative splicing can result from RNAs w/ different exon combinations
^1669869758298

Same gene makes calcitonin in the thyroid and ---::calcitonin-gene related peptide (CGRP) in the hypothalamus; If you splice together 1 2 3 4, you get calcitonin in thyroid but if you splice 1 2 3 5 6, you get calcitonin-gene in brain; Ex of tissue specific alternative splicing
^1669869733384

Whether calcitonin works in the thyroid or the brain depends on ---::tissue specific factors that regulate the processing of the primary transcript; Different sets of exons = different proteins (if you get exome sequenced, you can learn what proteins you can produce)
^1669869733392

mRNA degradation::The protein-coding region of mRNA is flanked by 5' and 3' untranslated regions (UTRs); The presence of RNA-binding proteins (RBPs) at the 5' or 3' UTR influences the stability of the RNA molecule
^1669869733399

Exonuclease (bio)::Digestive enzyme that "eats" nucleic acids; RNA has protective 5' cap and poly-A tail to protect from degradation; RNAs that are longer lived typically have longer poly-A tails; RNA can start to degrade within hrs and may last a couple days (while DNA can stay around for ages)
^1669869758305

Micro RNA (miRNA)::Production of functional miRNA begins in nucleus; Formed from pre-miRNAs that are chopped up by dicer protein into 22 nucleotide long segments; Ends in cytoplasm w/ 22 nt RNA that functions to repress gene expression
^1669869733408

miRNA is loaded into ---::RNA induced silencing complex (RISC); RISC binds w/ miRNA in cytoplasm to degrade target mRNA
^1669869733416

RISC::RNA induced silencing complex; Binds w/ miRNA and then binds to mRNA to inhibit translation
^1669869733423

miRNAs are complementary to one region of ---::mRNA; RNA typically single stranded so where miRNA binds, RNA is then double stranded and cannot fit into ribosome; Binding inhibits translation; miRNA can also have hairpin loop that prevents translation
^1669869733430

Translation begins w/ which IF?::Eukaryotic initiation factor-2 (elF-2); Initiation factors are used for translation; Lowercase e = eukaryotic; eIF-2 activated after binding GTP (a phosphorylated eIF-2 *cannot* bind GTP); When eIF2 is *not* phosphorylated, translation occurs
^1669869733437

Activated eIF-2 binds to the 40S ribosomal subunit forming the ---::eIF-2/40S ribosome complex; Initiator complex then consists of eIF-2/40S ribosome complex, MET tRNA, and mature mRNA
^1669869733444

Unphosphorylated eIF-2 can allow for ---::translation to occur while phosphorylated eIF-2 cannot bind to GTP and thus cannot be activated to bind to 40S ribosomal subunit to form complex; Ex of translational control
^1669869733452

After you make a protein, there are still ways to ---::control it (post-translational control); If you phosphorylate a protein, this is a post-translational modification; More examples are glycoproteins/glycolipids (addition of lipid or sugar) and disulfide bridges (links between S atoms of two cystine amino acids to stabilize tertiary structure)
^1669869733459

Hydrolyzing GTP activities ---::translation; Phosphate and eIF-2 are released, 60S ribosomal subunit binds to initiator complex, and translation begins
^1669869733466

Gene expression can be controlled by factors that bind to translation ---::initiation complex; Ex: When eIF-2 is phosphorylated, translation is blocked (P + EIF2 + 40S ribosomal subunit = no translation); When eIF-2 is not phosphorylated, translation occurs (EIF2 + 40S subunit = translation occurs)
^1669869733473

Protein degradation::Proteins are produced and degraded continually in the cell; Lysosomes house proteases for nonspecific protein digestion; Proteins marked specifically for destruction w/ ubiquitin (degradation then occurs at proteasome)
^1669869733481

--- is mark on protein for degradation::Ubiquitin; Like ticket that indicates protein will be destroyed; Protein is taken to proteasome that can take polypeptide chain and break it into individual amino acids; This process involves energy (ATP --> ADP + Pi); Ubiquitin ligase adds ubiquitin
^1669869733487

The catabolic process of breaking down a protein is called ---::hydrolysis; When you break down a protein, you add water (opposite is dehydration synthesis, which removes water and builds things up)
^1669869733494

GLUT4 is a glucose transporter that is stored in ---::vesicles; A cascade of events that occurs upon insulin binding to membrane receptor causes GLUT4-containing vesicles to fuse w/ membrane so glucose can enter cell; Glucose is too big and polar to get through itself
^1669869733501

When insulin binds on receptors, the ---::glucose transporter protein (GLUT4) opens and allows glucose to enter cell in vesicle
^1669869733509

People w/ diabetes have low energy and are ---::dehydrated; Dehydrated bc blood has more sugar outside than inside (so water leaves cells into blood); Low energy bc insulin can no longer trigger receptors so glucose cannot enter cell to make energy
^1669869733516

