[[2022-11-22]], [[2022-11-29]]

1. Structure is directly related to control of [[Gene Expression]]
	- [[DNA]] wound around histone proteins to form nucleosomes
		1. There are no nucleosomes in prokaryotes
		2. Nucleosomes may block access to promoter
		3. It is possible to transcribe nucleosome region of [[DNA]] (can read it while wrapped around histone [[Protein]])
			- Histones can be modified to result in greater condensation
				1. These modifications either allow access to DNA or they don't 
				2. Access may depend on methylation
					- Methylation once thought to play a major role in gene regulation
						- ---
					- Methyl group
						1. -CH3
					- Acetyl group
						1. CH2CH3
						2. Acetyl groups are bigger than methyl group
						3. If two carbons, acetyl group. If one carbon, methyl group
					- Histones can be methylated or acetylated
						1. In histones that are methylated, [[DNA]] cannot be read
							- Too tightly packed
						2. Histones that are acetylated are accessable
							- Provides necessary space for DNA to be read
						3. DNA itself can become methylated as well
							- Original strands of [[DNA]] are methylated whereas newly synthesized DNA strands are not
								1. This is how [[Cell]] knows if [[DNA]] is wrong and can correct newly synthesized DNA in replication process
						4. If histones methylated - inactive region
							- Heterochromatin
						5. If histones acetylated - active region
							- Euchromatin is more spread out bc it has two carbons
						6. ---
				3. [[DNA]] is folded around histone proteins to create nucleosome complexes. These nucleosomes control the access of proteins to the underlying DNA. When viewed through an electron microscope, the nucleosomes look like beads on a string. 
				4. Nucleosomes can slide along [[DNA]]. When nucleosomes are spaced closely together (top), transcription factors cannot bind and [[Gene Expression]] is turned off. When the nucleosomes are spaced far apart (bottom), the DNA is exposed. Transcription factors can bind, allowing [[Gene Expression]] to occur. Modifications to the histones and DNA affect nucleosome spacing.
				5. Histone proteins and [[DNA]] nucleotides can be modified chemically. Modifications affect nucleosome spacing and [[Gene Expression]].
					- If methylated, tight. If acetylated, looser.
					- Epigenetics
						1. Epigenetic changes to the chromatin may result from. . .
							- Development
							- Environmental chemicals
							- Drugs/Pharmaceuticals
							- Aging
							- Diet
						2. You accumulate more methylation over time based on your lifestyle
							- Environment puts meth tags on DNA
							- Keeps some genes from being accessed and allows others to be accessed 
								1. Times you want genes to turn on and times you don't want genes to turn on
						3. Epigenetic changes may result in. . .
							- [[Cancer]]
								1. Uncontrolled [[Cell]] growth
								2. Cells that blow through checkpoint in cell cycle start to have trouble as bad info is copied 
								3. 90% of cancers are in epithelial tissues
							- Autoimmune disease
							- Mental disorders
							- Diabetes 
					4. ---
	- Histone acetylases
		1. Some coactivators have been shone to be these
		2. ---
2. Control of transcription is more complex in eukaryotic regulation
	- Major differences from prokaryotes
		1. Eukaryotes have [[DNA]] organized into chromatin
			- Complicates protein-[[DNA]] interaction
		2. Eukaryote transcription occurs in nucleus
		3. Amount of DNA involved in regulating eukaryotic genes much larger
	- General transcription factors
		1. Necessary for the assembly of a transcription apparatus and recruitment of RNA pol II to a promoter
		2. TFIID recognizes TATA box sequences
		3. ---
	- ---
	- Enhancer = DNA sequence that promotes transcription
		1. Upstream of where the code that is transcribed is
			- Interact w/ proteins known as activators that interact w/ transcriptional machinery
		2. Each enhancer made up of short [[DNA]] sequences called distal control elements
		3. This interacting creates a region that facilitates transcription of one gene but by limit transcription of others
			- Crowded out
	- Activators = Bound to the distal control elements of enhancers and interact w/ mediator proteins and TFs
		1. Two different genes may have same promoter but different distal control elements, enabling differential [[Gene Expression]]
		2. Nearly every eukaryotic gene represents a unique case
		3. Greater flexibility to respond to many signals
		4. Virtually all genes that are transcribed by RNA pol II nee the same suit eof general factors to assemble an initiation complex
		5. ---
	- Transcription factors ---
3. Eukaryotic Posttranscriptional Gene Regulation
	- Control of gene expression usually involves the control of transcription initiation
	- Mature mRNAs contain 5' G-cap, 3' poly-A tail, and introns spliced out
	- [[Gene Expression]] can also be controlled after transcription w/. . .
		1. Alternative splicing
			- Introns are spliced out of pre-mRNAs to produce the mature mRNA
			- Tissue specific alternative splicing
				1. Can create one RNA w/ different exon combinations
				2. Same gene makes calcitonin in the thyroid and calcitonin-gene related to ---
				3. ---
				4. If you splice together 1, 2 ,3, 4, you get calcitonin in thyroid. If you splice together 1, 2, 3, 5, 6, you get calcitonin-gene in the brain
			- Different sets of exons = different protein
			- If you get your exome sequenced, you can learn what proteins you can produce
		2. mRNA degradation
			- The [[Protein]]-coding region of mRNA is flanked by 5' and 3' untranslated regions (UTRs)
			- The presence of RNA-binding protein (RBPs) at the 5' or 3' UTR influences the stability of the RNA molecule
			- Exonuclease = digestive enzyme that "eats" nucleic acids; RNA has protective 5' cap and poly-A-tail to protect from degradation
				1. RNAs that are longer lived typically have longer tails
				2. Length of poly-A-tail influences mRNA lifespan
					- RNA can start to degrade within hours and may last a couple day (unlike [[DNA]] that can stay around for ages)
		3. miRNA
			- Production of a functional miRNA begins in the nucleus
			- miRNAs are 22 nucleotides long
			- Formed from pre-miRNAs that are chopped up by the dicer [[Protein]]
				1. Dicer creates the 22 nucleotide pieces
			- Ends in the cytoplasm w/ 22 nt RNA that functions to repress [[Gene Expression]]
			- miRNAs are complementary to one region of mRNA
				1. RNA is typically single stranded
				2. Where miRNA binds, the RNA is then double stranded and cannot fit into ribosome
			- miRNA loaded into RNA induced silencing complex (RISC)
				1. RISC binds w/ miRNA in cytoplasm to degrade target mRNA
			- miRNA can have hairpin loop that prevents translation
			- Binding of miRNA to mRNA inhibits translation bc ribosome cannot read it 
			- See graph of process ---
	- Translation begins w/ eukaryotic initiation factor-2 (eIF-2)
		1. Initiation factors are used for translation
		2. A lowercase e in front indicates eukaryotic
		3. eIF-2 is activated after binding to GTP (a phosphorylated eIF-2 blocks translation)
			- When eIF2 is not phosphorylated, translation occurs
		4. ---
		5. This is a translational control
4. After you make a protein, there are still ways to control it (post-translational control)
	- If you phosphorylate a protein, that is a post-translational modification
	- Glycoproteins = [[Protein]] + sugar (post translational modification)
	- Disulfide bridges
		1. Post translational modification
		2. Cystine
		3. ---
	- Protein degradation
		1. Proteins are produced and degraded continually in the [[Cell]]
		2. Lysosomes house proteases for nonspecific [[Protein]] digestion
		3. Ubiquitin is mark on protein for degradation
			- Like ticket that indicates protein will be destroyed
			- Ubiquitition involves the binding of ubiquitin. Protein is then taken to proteasome that can take polypeptide chain and break it into individual [[Amino Acids]].
				1. This process involves some energy, so [[ATP]] --> ADP + Pi
				2. Ubiquitin ligase adds ubiquitin
				3. See graph of process ---
		4. The catabolic process of breaking down a protein is called *hydrolysis*
			- When you break down a [[Protein]], you add [[Water]]
				1. Opposite is dehydration synthesis (removes water) to build things up
	- GLUT4 is a glucose transporter that is stored in vesicles
		1. A cascade of events that occurs upon insulin binding to a receptor in the plasma membrane causes GLUT4-containing vesicles to fuse w/ the plasma membrane so that glucose may be transported into the [[Cell]].
		2. Glucose is too big and polar to get through membrane by itself
		3. Insulin binds on receptors, opening the glucose transporter protein and allowing glucose to enter the cell
		4. People w/ diabetes have low [[Energy]] and are dehydrated.
			- Dehydrated bc the blood has more sugar outside than inside the cell (so [[Water]] leaves the cells into the blood)
			- Low energy bc insulin can no longer trigger receptors so glucose cannot get inside [[Cell]] to make [[Energy]]