[[2022-11-22]]

1. Prokaryotic cells often respond to their environment by changes in [[Gene Expression]]
2. Operons are *only* found in prokaryotic 
	- NOT in eukaryotes
	- Operons are regions on chromosome that regulate for similar proteins
	- Genes involved in the same metabolic pathway are organized in operons
3. Have proteins that activate transcription and proteins that repress transcription
	- Induction = [[Enzymes]] for a certain pathway that are produced in response to a substrate
	- Repression = capable of making an enzyme but does not control transcription initiation
	- Positive control = increases frequency of initiation of transcription
		1. Activators enhance binding of RNA pol to promoter
			- Facilitate transcription
		2. Effector molecules can enhance or decrease
	- Negative control - decreases frequency of transcription
		1. Repressors bind to operators in [[DNA]]
		2. Allosterically regulated
		3. Respond to effector molecules - enhance or abolish binding to [[DNA]]
	- TRP operon
		1. Genes for the biosynthesis of tryptophan
		2. The operon is not expressed when the [[Cell]] contains sufficient amounts of tryptophan
			- Plenty found in environment
		3. *trp* repressor is a helix-turn-helix [[Protein]] that binds to the operator site located adjacent to the trp promoter
			- When it binds, RNA pol does not have normal landing place
		4. The five genes that are needed to synthesize tryptophan in E. coli are located next to each other in the trp operon
			- When tryptophan is plentiful, two tryptophan molecules bind the repressor [[Protein]] at the operator sequence. This physically blocks the RNA polymerase from transcribing the tryptophan genes
			- When tryptophan is absent, the repressor protein does not bind to the operator and the genes are transcribed
			- 5 start codons and 5 stop codons for 5 different proteins that need to be made
		5. *trp* operon is negatively regulated by *trp* repressor [[Protein]]
			- *trp* repressor binds to the ---
			- ---
	- Glucose repression
		1. Preferential use of glucose in the presence of other sugars
			- Mechanism involves activator protein that stimulates transcription
			- Catabolic activator [[Protein]] (CAP) is an allosteric protein w/ cAMP as effector
				1. If you cleave off phosphate of [[ATP]], you get ADP. If you cleave off another, you get AMP. If you are really really low on energy, cAMP is created.
					- cAMP means that [[Cell]] is very low on [[Energy]]
					- Must get glucose into [[Cell]] to reduce level of cAMP
				2. cAMP binds to allosteric site of CAP and changes its shape
					- This induces transcription
			- Level of cAMP in cells is reduced in the presence of glucose so that no stimulation of transcription form CAP-responsive operons takes place
		2. Induced exclusion - presence of glucose inhibits the transport of lactose into [[Cell]]
			- Cells prefer to use glucose over all other sugars 
		3. When glucose levels fall, E. coli may use other sugars for fuel but must transcribe new genes to do so. As glucose supplies become limited, cAMP levels increase. This cAMP binds to the CAP [[Protein]], a positive regulator that binds to an operator region upstream of the genes required to use other sugar sources.
			- The lacZ operon uses lactose
			- When cAMP + CAP binds to mRNA, RNA pol is activated and transcribes operons such as lacZ that allows cell to get more energy
				1. If glucose used up, can take lactose and other sugars 
		4. Lac operon - negatively regulated by a repressor [[Protein]]
			- Lac repressor binds to operator ---
			- ---
			- If lac repressor is bound, there is no lactose present. If there is lactose present, it may allow RNA pol to be read. But this also depends on present of glucose (if glucose, cell will not use glucose)
				1. Decisions must be made depending on resources
				2. Transcription of the lac operon is carefully regulated so that its expression only occurs when glucose is limited and lactose is present to serve as alternate [[Energy]] source