[[2022-11-03]], [[2022-11-08]]

1. [[The Genetic Material]]
	- Meselson and Stahl (late 1950s) investigated DNA replication, considering 3 possible mechanisms
		1. Conservative
			- [[DNA]] opens up and creates new copy
		2. Semiconservative
			- DNA opens up and partially stays together
		3. Dispersive
			- Separate two strands and go back and forth between new and old strand
		4. ---
		5. Meselson and Stahl took radioactive form of N, N15
			- Nitrogen is in nitrogenous bases (G, T, A, C)
			- Bacterial [[Cell|cells]] were grown in a heavy isotope of nitrogen, N15
				1. All the [[DNA]] incorporated N15
				2. N15 has mass of 15 whereas typical N has mass of 14
				3. Binary fission occurs once every 20 mins
			- Bacterial [[Cell|cells]] then switched to media containing N14
				1. DNA was extracted from cells at various time intervals
				2. [[DNA]] from different time points analyzed for ratio of N15 to N14 it contained
				3. At time 0, weight heaviest w/ only N15. At time 20, a little lighter. At time 40, lighter, etc.
			- After 1 replication, DNA consisted of 14N-15N hybrid molecule
			- After 2 replications, DNA contained 2 types of molecules (half was N14-N15 hybrid and half composed of N14 alone)
				1. Centrifuged to find ratios (heaviest closer to bottom)
			- Conclusion: the process of [[DNA]] replication is semiconservative 
				1. When strands open up, both strands used as template for a newly synthesized strand
				2. See graphic ---
	- Each strand of DNA acts as a *template* for the synthesis of a new strand
		1. If strand has A, T must be created as complement. If strand has G, complement must be C.
		2. Parental [[DNA]] strands require [[Enzymes]] to do the copying and nucleotide triphosphates to build the new strands
			- DNA polymerase: new strands of DNA are synthesized by this enzyme
				1. Allows for elongation as new strands are created
			- Termination: replication is terminated differently in [[Prokaryotic Cell|prokaryotes]] (circular chromosomes) and eukaryotes (linear chromosomes)
	- [[Prokaryotic Cell|Prokaryotic]] [[DNA]] replication
		1. Origin of replication
			- Characteristics of DNA in origin of replication
				1. Lots of AT base pairs bc AT only has 2 H bonds vs the 3 H bonds of GC base pairs
					- Makes DNA easier to open up
			- [[DNA]] polymerase only works from 5' to 3' in one direction. Nucleotide always added to 3' end as DNA elongates
			- DNA polymerases bind to an origin and replication proceeds in both directions around the chromosome
				1. Point of origin on one side of circle and point of termination of other side of circle. [[DNA]] polymerase works around the circile from origin to termination
				2. Replisomes = group of proteins that do polymeration. 
					- DNA polymerase is critical enzyme in this group
					- Helicase = unzips the double helix bc DNA polymerase must only read one strand
					- Primase enzyme = adds a short RNA primer sequence
						1. In order to copy, DNA polymerase must have a little primer (short piece of RNA)
						2. It is from this temporary primer that DNA polymerase creates new strand from 
					- [[DNA]] polymerase III (pol III) = enzyme that adds nucleotides one at a time to the 3' end of the new (daughter) strand of [[DNA]]
						1. Polymerase that does most of the work
						2. Can do 1000 bases a second in [[Prokaryotic Cell|prokaryotes]]
						3. The incoming nucleotide is triphosphate. Phosphates cleaved off to get [[Energy]] to make it work. Binds after two phosphates are cleaved off.
						4. Creates phosphodiester bonds to create sugar-phosphate backbone
		2. [[DNA]] replication is semidiscontinuous
			- Pol III can only add nucleotides to the 3' end of the new strand, but [[DNA]] strands are antiparallel
				1. Pol III only goes from 5' to 3' while adding these nucleotides
				2. How do you add nucleotide to strand that is going in the opposite direction?
					- Leading strand
						1. Synthesized continuously (in the same direction that the replication fork is moving)
					- Lagging strand
						1. Synthesized discontinuously creating Okazaki fragments
						2. Cannot be done continuously bc in opposite direction of pol III
					- Helicase unzips [[DNA]] to get replication fork. Pol III can continuously follow leading strand and only takes one primer. Lagging strand requires multiple fragments of primer. Then you have to go back and clean out the temporary RNA primer (RNAs are *always* temporary). 
						1. In the replisome, helicase is like little machine that opens up the DNA
							- Creates 3' - 5' strand and 5' - 3' strand
							- 5' to 3' is leading strand bc it goes in direction that [[DNA]] polymerase can follow continuously
							- Lagging strand is 3' to 5' and is organized into Okazaki fragments (segments of RNA primer that allow 5' to 3' DNA polymerase to work)
							- Primase creates the little bit of RNA to create little bit of doublestandedness so polymerase can start replication
				3. While being replicated, the single strands innately want to come together w/ other strand but we want them to stay apart to continue replication
					- Must temporarily insulate (or coat) [[DNA]]
					- Single stranded binding protein puts insulation on each of the strands to prevent annealing (coming together w/ H bonds)
			- [[DNA]] polymerase
				1. RNA polymerase makes primer and DNA polymerase extends primer from 5' to 3'
				2. Matches existing DNA bases w/ complementary nucleotides and links them 
				3. All have several common features. . .
					- Add new bases to 3' end of existing strands
					- Synthesize in 5' - 3' direction
					- Requires primer of RNA
				4. Proofreading 
					- [[DNA]] polymerase can do 3' to 5' exonuclease capability to profred
						1. If wrong base is added, can backup, cut nuclease out (exonuclease ability) then continue replication from 5' to 3'
			- [[Prokaryotic Cell|Prokaryotic]] [[DNA]] Replication Sum
				1. [[Enzymes]] for DNA replication are --- 
				2. ---
				3. See graphic "The replisome and additional [[Enzymes]] involved in DNA replication"
				4. Sliding clamp = holds [[DNA]] polymerase into right position to copy as it moves down the strand
				5. Single stranded binding proteins = especially important for lagging strand; SSB
				6. Helicase = unzips DNA
				7. Primase = adds temp RNA primer for DNA polymerase to start
				8. Three [[DNA]] polymerases in total
					- Pol I = reads doublestrandedness and cuts (exonuclease abilities) RNA primer out, replacing it w/ DNA
						1. Also does proofreading
						2. Acts on lagging strand to remove primers and replace them w/ DNA
					- Pol II = involved in [[DNA Repair]] process
					- Pol III = does most of copying; main replication enzyme
					- Little nick (break) in sugar phosphate backbones that DNA polymerase cannot fix after primer
						1. [[DNA]] ligase = reads along doublestranded DNA and wherever there is break, ligase can ligase (or link together) the backbone
							- Final worker that completes DNA synthesis 
					- All 3 have 3' - 5' exonuclease activity for proofreading
					- DNA pol I has 5' - 3' exonuclease activity for RNA primer removal 
					- Exonuclease = "ase" indicates enzyme; works on nucleic acids; exo indicates working on end
					- Endonuclease = enzyme that can work in middle on nucleotides 
				9. E coli was studied to learn about prokaryotic DNA replication
2. [[Eukaryotic Cell|Eukaryotic]] [[DNA]] Replication
	- The larger size and complex packaging of eukaryotic chromosomes means they must be replicated from *multiple origins*
		1. [[Prokaryotic Cell|Prokaryotic]] DNA replication takes about 20 mins
			- Drosophila (fruit fly) has genome much bigger than bacteria but [[Cell]] division takes 8 mins bc of multiple origins of replication on linear chromosome
				1. Starts replicating at multiple places
				2. One spot goes in one direction and other starts in opposite direction so that they meet up
			- Can start at multiple points whereas prokaryotes can only start on one part of circular chromosome
	- Initiation phase of replication requires more factors ---
		2. ---
	- Overall process is pretty similar to prokaryote replication except that chromosomes are linear in eukaryotes 
		1. Problem = [[DNA]] polymerase needs primer to start copying. Removed primer after copying cannot be replaced.
			- How do you copy ends of chromosomes (telomeres)?
				1. If you don't copy ends, your chromosomes get shorter and shorter until you get to [[Protein]] instructions and those would be interfered w/ 
			- Solution = another [[Enzymes|enzyme]] responsible for restoring ends of chromosomes
				1. ---
				2. Telomerase = builds extension onto 3' overhang at end of leading strands
					- Critical to maintain ends of chromosomes
					- Aging is associated w/ decreased associated telomerase activity
						1. Environment, lifestyle, and [[Genetics]] all play a part as well
