[[2023-04-17]], [[2023-04-19]], [[2023-04-21]], [[2023-04-24]]

1. DNA Fingerprinting
	1. Lab method for graphically characterizing an individual's DNA
	2. Used in forensics 
	3. Can determine paternity
		1. If you don't know which bull is the sire
	4. Process
		1. Take a small sample of blood (or other tissue)
		2. Then, DNA is extracted
		3. Next, DNA is chemically copied (using PCR - polymerase chain reaction) and placed on a gel
			1. PCR has three main steps using a thermal cycler
				1. Denaturation step
					1. Double strand is broken (94C)
				2. Annealing step
					1. Primers bind to a specific section of the DNA (60C)
				3. Elongation/Extension step
					1. Nucleotides are added (72C)
				4. This process tends to take two and a half to three hours (runs again and again)
					1. lab gene = Interferon Alpha
		4. Sorted by electrophoresis
			1. Method that separates macromolecules (nucleic acids or proteins) on the basis of size, electric charge, and other physical properties 
			2. Shorter or smaller sections of DNA will move farther than longer sections 
			3. When looking at a heterozygote, you will see two different bands for the two different alleles 
			4. ![[Pasted image 20230417103138.png]]
				1. You can see which genes children have from which parents 
				2. The green zygo, cte is not genetically related in this example
			5. Results in a banding pattern resembling a bar code
				1. Pattern of bands represent the presence of specific alleles at loci
	5. DNA is polymorphic
		1. At least two alternative alleles occurring in a pop
			1. Can determine if homo or hetero
	6. Uses
		1. When using a multiple sire pasture system (vs AI or hand mating), DNA fingerprinting allows you to see which bull is able to impregnate more females by natural cover
		2. You can identify parents more accurately
			1. Avoids faulty pedigrees
		3. You can predict hybrid vigor
			1. *Less related* individuals will have greater hybrid vigor 
			2. ![[Pasted image 20230417103748.png]]
				1. You can visually see which ones share more alleles than others
		4. Makes selection more accurate, which brings about faster change of the population
2. Current Genetic Evaluation
	1. ![[Pasted image 20230417103908.png]]
	2. Currently, genetic merit is evaluated in the form of EDPS
		1. EPDs are computed from *phenotypes* of individuals and its relatives 
	3. But now that we can access genome, we can add another layer of info to improve selection (MAS)
	4. Marker Assisted Selection
		1. The selection for specific alleles using *genetic markers*
			1. If you use this, you carry out marker assisted management (MAM)
		2. Genetic Marker
			1. Detectable genes or DNA fragments used to identify alleles at a linked locus
				1. Detectable means we can identify different versions of alleles
				2. Some alleles are detectable because they produce easily distinguishable phenotypes
			2. Genetic markers now detect using DNA fingerprinting
			3. Human ex:
				1. BRAC1 or BRAC2 genes 
		3. Linkage Analysis 
			1. We link alleles to the presence or absence of certain phenotypes 
			2. Mathematical process that uses info from specially bred populations to determine whether two loci are linked and how closely
		4. The best "markers" are the alleles of interest themselves
		5. Specific markers 
			1. RFLP (restricted fragmental length polymorphism)
				1. Steps:
					1. Extract DNA
					2. Use restriction enzyme to cut the DNA
						1. Type of enzyme depends on the size and type of fragment you want
					3. Use Gel electrophoresis, to visualize DNA and identify the alleles 
				2. If heterozygous, produces two bands
					1. If homo, produces one of two different bands (ex higher for rec and lower for dom)
			2. Microsatellites
				1. Small pieces of DNA that repeat
					1. Deals w/ tandem repeats
				2. Ex graphic![[Pasted image 20230419102359.png]]
					1. CTT is repeating in this example
					2. Animals w/ eight repeats may be better than those with seven or nine 
				3. You can associated differences in performance of animal w/ amount of repeats they have
				4. Microsatellites are good for genetic diversity studies and comparing breeds but hard to genotype
			3. SNP (single nucleotide polymorphism)
				1. A single base change
				2. Ex: 95% of pop has A at a specific location. 5% of pop has G a that same location. 
				3. SNPs becoming more popular than microsatellites because it is easier to genotype
					1. Can be used for everything
				4. Can create GEBVs (genomic estimated breeding values) based off SNPs
			4. CNV (copy number variation)
				1. Structural variations that can be due to duplication, deletion, or some other change that affects the DNA's variation
				2. Especially linked to health
					1. Ex: Disease genomics 
					2. More reliable than other markers in this area
		6. More difficult to use MAS in polygenetic traits
			1. This is because multiple genes affect the trait so you must establish multiple associations
				1. Performance should not be affected much by just one gene
				2. Identifying effect of one gene can be difficult 
			2. Ex: growth rate, racing ability
			3. Epistatic effects also complicate things
			4. Major gene effect
				1. One gene may account for 50% of variation in a pool of thousands of genes
					1. A single very influential gene 
				2. If you develop marker for that one gene, then your level of accuracy will be higher 
		7. Geneticists estimate linkage distance by examining the rate at which genes segregate at two loci combined
			1. Crossing over
				1. If distantly linked
				2. Crossing over may occur and marker will "code" for an alternate allele
	5. A more realistic approach to genetic evaluation 
		1. ![[Pasted image 20230419103617.png]]
		2. Can use both phenotypes and DNA tests on the individual and its relatives
		3. The NCE can help produce marker-adjusted EPDs upon which selection decisions can be made
3. Gene Transfer
	1. Transplantation of specific genes from one individual to another 
		1. DNA is universal across organisms 
		2. Ex: goat that produces human lyosine (improves shelf-life of milk but not naturally found in goats)
	2. Molecular Pharming
		1. Can take a non-native gene and get it expressed in another organism
	3. Involves transfer of genes that code for important proteins from domestic animals to bacteria
		1. Bacteria can be genetically altered to manufacture the protein cheaply
		2. Ex: bacteria and insulin 
	4. Slow process
		1. Only one copy of gene is transferred at one time
		2. Takes generations to spread that gene throughout pop
		3. Gene could end up anywhere in the genome
			1. With advance of tech, things are getting more and more precise
	5. Ex: Bt corn
		1. Alleviates susceptibility to corn borers insects 
		2. Cry gene kills the insects when they attack
	6. Transgenic
		1. An individual that has received genetic material by gene transfer
		2. Transgenic dairy cows can produce human insulin
		3. Transgenic animals can obtain genes for disease resistance
			1. Improvement from a welfare standpoint
	7. Easier to do in plants than in animals
		1. Many traits are simply inherited and simply regulated
		2. Pest resistance and insecticide resistance
4. Genomic Selection
	1. What is *genomics*?
		1. The study of how the genome (DNA) of any species is organized and expressed as traits
			1. *Genome* = totality of DNA that an individual has
			2. Can see how DNA differs between species 
		2. New technologies allow examination of an organism's entire genome rather than just one gene at a time
		3. Livestock and poultry genomes sequenced to understand how various genes function 
			1. *Functional genomics*
	2. Applications in genetic improvement
		1. Can find easily detected genetic differences among animals
			1. Used to develop *SNPs (single nucleotide polymorphism) panels* to detect these differences
			2. Relate SNP differences to productivity differences
		2. Can rank animals on their economic merit
			1. Allows us to use best animals as parents of the next generation by creating *GEBVs (genomic estimated breeding value)*
	3. Federal support for genomics 
		1. Cattle
		2. Sheep
		3. Swine
		4. Poultry
		5. Horses
		6. Aquaculture (fish and other aquatic animals)
	4. How do we use genomics?
		1. Can identify DNA sequences associated w/ disease resistance and production traits
		2. Animals can be evaluated as soon as DNA is obtained (even before birth)
			1. Best animals to be parents can be determined earlier and more accurately
	5. Dairy cattle selection **before** genomics
		1. Slow!
			1. Progeny testing for production traits took 3-4 years from insemination
			2. Bull had to be at least five years old before first evaluation available
		2. Expensive!
			1. Progeny testing costed $25-50000/bull
			2. Only one in 8-10 bulls graduated (pass) progeny test
			3. At least $200,000 invested in each active bull
		3. Genomic selection has reduced these issues
			1. Shorter Gi
			2. Additional layer of info from genome
			3. More accurate selection decisions 
	6. Genetic markers 
		1. Segments of DNA at unique physical locations in the genome that vary sufficiently (polymorphic) between individuals 
			1. Inheritance can be tracked through families and followed in a region across generations 
		2. Markers not required to be part of a gene
			1. Markers can be very close to the gene of interest
		3. SNPs are the markers of choice
			1. ![[Pasted image 20230421102540.png]]
			2. Need lots - 3 million in the genome
	7. Whole Genome
		1. Ex: BovineSNP50
			1.  ![[Pasted image 20230421102634.png]]
			2. This is a SNP chip that can be used to scan 50,000 locations in the gene
		2. Can do two things with the whole genome
			1. GWAS (genome wide association study)
				1. Associates the info in the genome w/ a particular trait
					1. Sees which region of genome is associated w/ differences 
				5. Ex: chromosome 14 is tied to milk production in dairy cattle
					1. Gene = DGAT-1
					2. GWAS allows you to see that this part of the genome is linked w/ milk yield
				2. Fine mapping
					1. First find general megabase region then fine map by narrowing down the region 
				3. Gene discovery
					1. Then you find which gene is responsible 
				4. Allele variation
					1. Then you look at the type of variation
				5. animalgenome.org
					1. Tells you what megabase a gene is within 
					2. Then you have to refine (fine mapping) to kilobase then to base pair 
			2. WGS (whole genome selection)
				1. This is what most farmers use
				2. Estimates breeding values to generate estimates of performance 
		3. 770K Chip
			1. ![[Pasted image 20230421103503.png]]
				1. Illumina Bovine HD
			2. A higher SNP density panel than the 50K chip
				1. 770,000 Single Nucleotide Polymorphisms (SNPS)
			3. Should allow predictions to be less breed-specific 
	8. Summary
		1. Genomics is revolutionizing animal breeding
		2. Genomic selection used extensively in dairy cattle breeding
		3. High quality genotypes support detection of parentage and other errors
		4. International collaboration has been important for the success
			1. Shared data
5. Gene Editing
	1. Meganucleases
	2. Zinc Finger Nucleases (ZFNs)
		1. Deals w/ nine nucleotide target sites
		2. High editing failure rate (0.89)
	3. Transcription-Activator Like Effector Nucleases (TALENs)
		1. More effected than ZFNs or meganucleases
			1. Can get a bigger chunk (16-17 nt target sites) as well
		2. Primary tech before CRISPR
		3. Proteins w/ two domains
			1. One recognizes region to cut
			2. Other allows it to make specific cut
		4. High editing failure rate (0.79 - but lower than ZFN)
	4. CRISPR/Cas9
		1. Cas9 is the guide protein
			1. CRISPR is what makes the specific cut
		2. Cas9 nuclease induces double strand breaks
		3. Can have NHEJ or HDR make changes
		4. Can do insertions (unlike prior tools) as well as deletions
		5. Lower editing failure rate (0.37)
	5. See graphics ---
	6. Applications of genome editing in livestock species
		1. Disease Resistance
			1. Ex: pigs w/ PRRS virus resistance were generated by knocking down the scavenger receptor CD163 
				1. PRRS is estimated to cost pig breeders more than 6 mil each day
				2. Porcine Reproductive and Respiratory Syndrome
			2. Ex: cattle resistant to mycobacterium bovis infection may be created by genome editing 
		2. Improved Performance
			1. Ex: Myostatin (MSTN) results in double muscling due to hyperplasia and hypertrophy
				1. Meat output advantages
			2. Ex: The main b-lactoglobulin gene, which encodes a key allergenic milk protein in cattle, may be knocked out using ZFN
		3. Animal Welfare
			1. Ex: Gene editing has produced horn-free Holsteins so traditional methods to remove horns are not necessary
				1. Uses TALEN-based gene editing tech
		4. Biomedicine
			1. Ex: Gene-edited pigs that may be utilized in biomedical applications (such as organ transplants) by employing nucleases to delete certain genes such as GGTA1-gene that encodes a sugar that functions as an epitope on the surface of pig cells and is critical to success of xenotransplantation 
		5. Improving heat tolerance
			1. Ex: "Slick" cattle have shorter hair that helps then stay cooler in higher temps 
				1. CRISPR-Cas9 editing can replicate this Caribbean trait in American beef cattle
		6. Offspring sex
			1. Ex: It's possible to use CRISPR tools to insert DNA sequence into some breeds of bulls so that they only have male offspring, which produce more meat
		7. More examples of gene editing in livestock
			1. See graphic ---
		8. Know genome editing tools and applications